README.md

```diff
@@ -16,15 +16,53 @@ flux-accounting requires the following packages to build:
 
 ### Install Instructions
 
-You can install the dependencies required by flux-accounting (located in **requirements.txt**) with the following command:
+You can install the dependencies required by flux-accounting as well as the package itself to be recognized by Flux's command driver `flux(1)` with `make`.
+
+1. Set your **FLUX_INSTALL_PREFIX** environment variable to point to flux-core's installation target:
+
+```
+$ export $FLUX_INSTALL_PREFIX = ~/path/to/flux-core/install/
+```
+
+2. After building flux-core, you'll need to set **PYTHONPATH** to point towards flux-core's Python site-packages as well as the **PATH** to point to Flux's executable:
+
+```
+$ export PYTHONPATH = ~/path/to/flux-core/install/lib/python3.7/site-packages/
+$ export PATH = ~/path/to/flux-core/install/bin/
```

Comment:
Looks good, we just want to make sure we don't wipe out existing paths in `PATH`.

```suggestion
$ export PATH = ~/path/to/flux-core/install/bin/:$PATH
```

Reply:
We can also leverage the previously exported prefix to reduce duplication.

```suggestion
$ export PATH = ${FLUX_INSTALL_PREFIX}/bin/:$PATH
```